chat-plugins/info.js

```diff
@@ -328,7 +328,7 @@ var commands = exports.commands = {
 	dt: 'details',
 	details: function (target) {
 		if (!target) return this.parse('/help details');
-		CommandParser.commands.data.apply(this, arguments);
+		this.run('data');
```

Comment:
Why wouldn't that work if `/data` is monkeypatched? Is `commands !== CommandParser.commands`?